web/src/pages/index.page.tsx

```diff
@@ -51,7 +55,12 @@ const Home: NextPage = () => {
       <section id="how-it-works">
         <StyledBox>
           <Typography variant="h4">How it works</Typography>
-          <YoutubeEmbed embedId="yM8RrwQWeJc" />
+          <StyledCarousel autoPlay={false} navButtonsAlwaysVisible={true}>
+            <YoutubeEmbed embedId="yM8RrwQWeJc" />
+            <Image src={DemoPicture1} alt="mapping solutions to problems" />
+            <Image src={DemoPicture2} alt="comparing solutions" />
+            <Image src={DemoPicture3} alt="justifying and critiquing claims" />
```

Comment:
I wasn't sure how we wanted to name the images, so I just tried to summarize the contents for the `alt` property and give generic names for imports. Let me know what you'd like to see here.

On a separate note, I will squash all the commits into one after you approve the PR and before I merge it (if that's what you're looking for).

Reply:
1. re: naming; you should name the imported variables similarly to how you've named the image files, except use camelCase. E.g. comparingSolutions
2. I actually overlooked importing images and using them this way 😅 I think the way you've done it is better than [what I'm doing elsewhere](https://github.com/amelioro/ameliorate/blob/4ca37e071b884e3116414ccea6c90920ae6cfa0a/web/src/common/components/Layout.tsx#L29-L34). After reading the docs more closely, I'm pretty sure this way I'm doing elsewhere is actually treating the images as remote, but I didn't have to add them to the `remotePatterns` whitelist most likely because they're remotely accessed from the webserver itself. And since importing is the proper way to locally retrieve an image, I'm pretty sure nextjs will handle sending it to the client without needing to host it via the `public` directory
3. go ahead and squash the commits after you've addressed my suggestions. Until you're a member of the repository, you can't merge PRs yourself

Reply:
1. Will do.
2. I'm pretty sure I saw this in an example somewhere and decided to use it. Glad it was helpful. :)
3. Oops, that makes sense.

Reply:
regarding the danger of overwriting history - as long as you've committed changes at some point, you should be able to retrieve them via [git reflog](https://git-scm.com/docs/git-reflog). commits are immutable and will only be deleted during automatic git garbage collection if they're dangling (not pointed to by a branch or another commit) for ~2 weeks (if I understand the [docs](https://git-scm.com/docs/git-gc#Documentation/git-gc.txt-gcpruneExpire) correctly...).

I think this article https://blog.git-init.com/does-amending-a-commit-change-its-hash/ (and the previous one it links to) explains git commit storage really well.

still happy to pair on commit management if you want. feel free to ping me on discord